roles/install-maven/tasks/main.yaml

```diff
@@ -0,0 +1,28 @@
+- name: Install Apache Maven {{ maven_version }}
+  shell: |
+    wget "https://archive.apache.org/dist/maven/maven-3/{{ maven_version }}/binaries/apache-maven-{{ maven_version }}-bin.tar.gz"
```

Comment:
seems this url is unreachable from some IPs, such as China. Suppose someday we have VMs located in China, this job won't work. How about use another mirror url, or try more than one url.

Reply:
maybe we can just assume all the nodes are outside GFW, also there are some other urls such as google.com in `openlab-zuul-jobs`

Reply:
hmm, I just checked. the `archive.apache.org` is the old release URL, now it is no longer recommend to use, please use `www.apache.org` instead. that is the reason we cannot open the url  in browser I think, see:
http://www.apache.org/dev/mirrors#location

Reply:
emm, sounds reasonable. My concern is that HuaweiCloud ARM resources are now only provided inside GFW.  If we use them, we'll hit the issue. Leave a TODO here is OK currently.

Reply:
Thanks very much for kind suggestion. ;-)